base/atomics.jl

```diff
@@ -80,6 +80,8 @@ end
 
 Atomic() = Atomic{Int}()
 
+LOCK_CONFLICT_COUNT = Threads.Atomic{Int}(0);
```

Comment:
it looks like this is only mutated, not re-bound, so it could be const:
```suggestion
const LOCK_CONFLICT_COUNT = Threads.Atomic{Int}(0);
```